website/src/components/RespImage/index.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2020 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of the GNU Affero General Public License v3.0.
+ * You should have received a copy of the GNU Affero General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *
+ */
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+
+interface Source {
+  src: string;
+  media: string;
+}
+
+export interface RespImageProps {
+  children: React.ReactNode;
+  sources: Array<Source>;
+}
+
+const RespImage = ({ children, sources }: RespImageProps) => (
+  <picture>
+    {sources.length > 0 &&
+      sources.map(({ src, media }, i) => <source key={i} srcSet={src} media={media} />)}
+    {children}
+  </picture>
```

Comment:
I'm learning so much about this `picture` tag right now lol!

Reply:
what's the problem preventing us from using this component?

Reply:
@hlminh2000 It's pretty cool!
Looked kinda messy when I used it .... really I was just trying to finish the ticket more quickly...  but really I should just use it here. I'll do that and merge because it's the same code that's just sitting in the `Navbar` right now :)